src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -2,10 +2,23 @@ import React from 'react';
 import { Story, Meta } from '@storybook/react';
 import SearchBar from './SearchBar';
 
+import { within, userEvent, waitFor } from '@storybook/testing-library';
+
+import { expect } from '@storybook/jest';
+
 export default {
   title: 'Components/SearchBar',
   component: SearchBar,
   argTypes: {
+    value: {
+      type: 'string',
+    },
+    onSearch: {
+      action: true,
+    },
+    onChange: {
+      action: true,
+    },
```

Comment:
Set the argTypes for functions to `action:true` if you want to test if they have been called in interaction tests (see below).